mudlet-mapper.xml

```diff
@@ -5395,13 +5401,35 @@ function speedwalking( event, num )
   if num ~= mmp.currentroom then mmp.previousroom = mmp.currentroom end
   mmp.currentroom = num
   mmp.currentroomname = getRoomName(num)
+	
+	-- Try to track if we're flying or not, for Imperian wings
+	-- This is to avoid being &quot;off path&quot; if we FLY due to wings.
+	local madeflight=false
+	if gmcp.Char then
```

Comment:
Why are you checking for `gmcp.Char` but don't use it? You probably want to check for `gmcp.Room`.

Also, if the formatter works here, could you run it on this script as well?